### ISSUE TITLE
Adding FileVault 2 & new FileVault Options Payloads results in same PayloadUUID

### ISSUE THREAD

**Issue by apizz** [ProfileCreator/ProfileCreator#210]
Per this MacAdmin Slack thread (https://macadmins.slack.com/archives/C2N7MED7G/p1586963469028700), for whatever reason despite adding two different payloads, albeit related to FileVault, the payloads themselves end up having the same `PayloadUUID`.  They very definition of a UUID is that it is unique, so something in ProfileCreator is not correctly changing this when either of these payloads are added after the other.

**FileVault 2 Payload Before being Added**

![Screen Shot 2020-04-16 at 2 48 46 PM](https://user-images.githubusercontent.com/11789931/79517176-709c8680-801b-11ea-8ad8-22dfeff6edd5.png)

**FileVault Options Payload Before being Added**

![Screen Shot 2020-04-16 at 2 48 47 PM](https://user-images.githubusercontent.com/11789931/79517187-72fee080-801b-11ea-90a2-99e045dd4bb2.png)

**FileVault 2 Payload After being Added**

![Screen Shot 2020-04-16 at 2 48 58 PM](https://user-images.githubusercontent.com/11789931/79517216-89a53780-801b-11ea-91cf-efa7e1d2e34f.png)

**FileVault Options Payload After being Added**

![Screen Shot 2020-04-16 at 2 49 05 PM](https://user-images.githubusercontent.com/11789931/79517225-9164dc00-801b-11ea-8865-6ce005d08d6b.png)

**Comment by erikberglund** [comment]
Yes. This happens if they are added in this order but not if you add options first. And that is because the function finding existing payloaduuid for domains is looking for the prefix and not an exact match. I can't really tell you why it's that way and there probably is a good reason.. I have made a change to only the part where it's finding the PayloadUUID and hope that doesn't break anything else. I believe the reasons could be for domains that share settings, but have not verified. Anyway, added that fix to a future update.
